include/bamit/IntervalNode.hpp

```diff
@@ -131,7 +131,7 @@ class IntervalNode
     template <class Archive>
     void serialize(Archive & ar)
     {
-        ar(this->start, this->end, this->lNode, this->rNode, this->file_offset);
+        ar(this->start, this->end, this->lNode, this->rNode, static_cast<std::streamoff>(this->file_offset));
```

Comment:
Why is this not streampos?

Reply:
Cereal can't archive streampos, so I have to archive it as a streamoff.